src/openrct2/object/Object.cpp

```diff
@@ -322,6 +322,29 @@ std::unique_ptr<IStream> ObjectAsset::GetStream() const
     return {};
 }
 
+std::string VersionString(Version version)
+{
+    return String::Format("%i.%i.%i", std::get<0>(version), std::get<1>(version), std::get<2>(version));
+}
+
+Version VersionTuple(std::string version)
+{
+    if (version.length() == 0)
+    {
+        return std::make_tuple(0, 0, 0);
```

Comment:
```suggestion
        return {0, 0, 0};
```

Technically just this would work